Apps/AdminWebClient/src/ClientApp/src/components/core/View.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <v-content>
+    <div id="core-view">
+      <v-fade-transition mode="out-in">
+        <router-view />
+      </v-fade-transition>
+    </div>
+    <core-footer v-if="$route.name !== 'Maps'" />
+  </v-content>
+</template>
+
+<script>
+export default {
+  metaInfo() {
+    return {
+      title: 'Vuetify Material Dashboard',
```

Comment:
Should this be changed?

Reply:
Will change it once we start working on the UI